.travis.yml

```diff
@@ -6,13 +6,12 @@ env:
         - use_coveralls=true
     matrix:
         - CONTAINER_IMAGE=quay.io/nmstate/centos8-nmstate-dev:nmstate-0.2
-          testflags="--test-type integ --pytest-args='-x'"
+          testflags="--test-type integ --pytest-args='-x'
```

Comment:
@EdDev are you ok with this change?

Reply:
No, I am not. We need an NM release otherwise no one can consume our releases.